### PR TITLE
Create 08_vs_010.md

### PR DESCRIPTION
**Pull request by beckettsean**

### CODE REVIEW

**Comment by beckettsean** [comment]
@rkuchan @gunnaraasen ready for review

**Comment by beckettsean** [comment]
closes https://github.com/influxdata/docs.influxdata.com/issues/259

File: content/influxdb/v0.10/concepts/08_vs_010.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+The API, the schema, the data store, and the clustering algorithm have all been significantly altered and in incompatible ways.
+The InfluxQL syntax has remained largely the same, although there are functions which exist only in one or the other version, and a few functions are implemented differently in the two versions.
+
+This page aims to ease the transition to InfluxDB 0.10 for those more familiar with 0.8.
+It is not a comprehensive list of the differences, that would be prohibitively large.
+
+## Schema
+
+The most noticeable difference between InfluxDB 0.8 and InfluxDB 0.10 is the way series are defined and organized.
+In InfluxDB 0.8 a series had an explicit name that incorporated all the metadata about the value, similar to Graphite output.
```
Comment (rkuchan) [inline content/influxdb/v0.10/concepts/08_vs_010.md]: I think this sentence should also be present tense to go with the rest of the paragraph.

File: content/influxdb/v0.10/concepts/08_vs_010.md [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+
+In InfluxDB 0.8 a series has a name and one or more measured values.
+The values can be passed to functions or used in a `GROUP BY` clause.
+The series name is indexed but the values are not.
+In InfluxDB 0.10 a series has a measurement name, a tag set, and a field set.
+The measurement name plus the tag set are equivalent to the series name from InfluxDB 0.8.
+The field set in InfluxDB 0.10 is equivalent to the series values in InfluxDB 0.8.
+
+
+When designing a schema for InfluxDB 0.10 is it important to understand the differences between tags and fields.
```
Comment (rkuchan) [inline content/influxdb/v0.10/concepts/08_vs_010.md]: Nit: I'd prefer bullets here:

* Tags are indexed, fields are not. Both can be used as filters in the `WHERE` clause, but filtering by tags only requires a quick index lookup. Filtering by fields requires a full scan of all points on disk that match the other filters and is thus much less performant.
* Fields can be passed into functions but tags cannot.
* Tags can be used in a `GROUP BY` clause but fields cannot. (In InfluxDB 0.8 series values -- equivalent to fields -- can be used in the `GROUP BY` clause.)
* Measurement names and tags can be matched using regular expressions but regular expressions are not valid for matching field keys or values.

File: content/influxdb/v0.10/concepts/08_vs_010.md [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+
+## Data Retention
+
+In InfluxDB 0.8 data retention is handled by shard groups.
+Shard groups include a duration for each individual shard within the group and when the duration is exceeded, the shard may be dropped.
+A given series writes to the first shard group whose regular expression matches the series name.
+Shards and shard groups are directly exposed in the Admin UI and can be manipulated by an end user.
+
+In InfluxDB 0.10 it is no longer necessary to parse or define regular expressions to determine the retention of a particular point.
+Instead, data retention is handled by retention policies.
```
Comment (rkuchan) [inline content/influxdb/v0.10/concepts/08_vs_010.md]: Maybe link to the glossary entry for rps: /influxdb/v0.10/concepts/glossary/#retention-policy-rp

File: content/influxdb/v0.10/concepts/08_vs_010.md [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+Retention policies belong to databases and define the duration for each point written to the retention policy.
+When the duration is exceeded, the underlying storage file is marked to be dropped.
+A point is written directly to a particular retention policy, declared at write time.
+If no retention policy is explicitly provided with the write, the point is written to the default retention policy.
```
Comment (rkuchan) [inline content/influxdb/v0.10/concepts/08_vs_010.md]: `DEFAULT`

File: content/influxdb/v0.10/concepts/08_vs_010.md [hunk 5]
```diff
@@ -0,0 +1,171 @@
...
+In InfluxDB 0.10 shards and shard groups are purely internal and have no user-facing features or settings.
+
+
+## Points With Identical Timestamps
+
+In InfluxDB 0.8 a point is uniquely identified by the series name, timestamp, and sequence number.
+The sequence number is an internally generated ID that provides uniqueness in the event the series name and timestamp are identical.
+
+
+In InfluxDB 0.10 a point is uniquely identified by the measurement name, full tag set, and the timestamp.
```
Comment (rkuchan) [inline content/influxdb/v0.10/concepts/08_vs_010.md]: Maybe link to the glossary entries for measurement, tag set, and timestamp:

*  /influxdb/v0.10/concepts/glossary/#measurement
*  /influxdb/v0.10/concepts/glossary/#tag-set
*  /influxdb/v0.10/concepts/glossary/#timestamp

File: content/influxdb/v0.10/concepts/08_vs_010.md [hunk 6]
```diff
@@ -0,0 +1,171 @@
...
+
+In InfluxDB 0.8 the primary write method is a JSON protocol.
+In InfluxDB 0.10 the JSON write protocol is deprecated and replaced by the [line](/influxdb/v0.10/write_protocols/line) [protocol](/influxdb/v0.10/write_protocols/write_syntax).
+
+
+## API Endpoints
+
+InfluxDB 0.8 has multiple endpoints for the API, often encoding concepts such as the database and shard space in the HTTP endpoint.
+InfluxDB 0.10 simplifies the common API interactions, using a common endpoint `/query` for all queries, and `/write` for all writes, regardless of database or retention policy.
+The database and retention policy are passed as query string parameters in InfluxDB 0.9.
```
Comment (rkuchan) [inline content/influxdb/v0.10/concepts/08_vs_010.md]: 0.9 > 0.10

File: content/influxdb/v0.10/concepts/08_vs_010.md [hunk 7]
```diff
@@ -0,0 +1,171 @@
...
+
+In InfluxDB 0.10 the `CREATE CONTINUOUS QUERY` statement also controls CQ execution.
+By default, CQs run at the same interval as the CQ's `GROUP BY time()` interval, and the system calculates the query for the most recent `GROUP BY time()` interval.
+A new and optional `RESAMPLE` clause allows users to specify how often the CQ runs and the time range over which InfluxDB runs the CQ.
+
+## Clustering
+
+InfluxDB 0.10 follows a hybrid model for maintaining availability of the data while guaranteeing consistency of the metadata.
+See CEO and co-founder Paul Dix's [clustering blog post](https://influxdata.com/blog/influxdb-clustering-design-neither-strictly-cp-or-ap/) for more details.
+InfluxDB 0.10 supports hybrid nodes, data nodes, and [consensus nodes](/influxdb/v0.10/concepts/glossary/#consensus-node).
```
Comment (rkuchan) [inline content/influxdb/v0.10/concepts/08_vs_010.md]: Link to other glossary entries for hybrid nodes and data nodes:
* /influxdb/v0.10/concepts/glossary/#hybrid-node
* /influxdb/v0.10/concepts/glossary/#data-node

**Comment by rkuchan** [comment]
A couple small things. Otherwise looks good!

**Comment by beckettsean** [comment]
great feedback, all addressed

**Pull request merged**